src/store/createUser/reducer.js

```diff
@@ -0,0 +1,108 @@
+import {
+  SET_USERS,
+  CREATE_SUCCESS,
+  CREATE_START,
+  CREATE_FAILURE,
+  FORM_USER_TYPE_INPUT_VALUE,
+  FORM_DESCRIPTION_INPUT_VALUE,
+  FORM_USERNAME_INPUT_VALUE,
+  FORM_PASSWORD_INPUT_VALUE,
+  FORM_PASSWORD_REPEAT_INPUT_VALUE,
+  PASSWORD_LENGTH_ERROR,
+  PASSWORD_EQUAL_ERROR,
+  EMPTY_FIELDS_ERROR,
+} from './types';
+import { EMPTY_FIELDS_ERROR_MESSAGE } from '../login/constants';
+import { PASS_LENGTH_ERROR_MESSAGE, PASS_EQUAL_ERROR_MESSAGE } from './constants';
+
+const initialState = {
+  totalUsers: {
+    count: 0,
+    isLoading: true
+  },
+  currentPage: 1,
+  search: '',
+  itemsPerPage: 4,
+  lastLoadedPage: 1,
+  error: false,
+  isLoading: false,
+  hasNextPage: true,
+  users: [],
```

Comment:
Remove these fields, expects error from createUser reducer as it does not relate to creating user logic